tests/Unit/BlackCard/BlackCardRelationshipTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Tests\Unit\BlackCard;
+
+use App\Models\BlackCard;
+use App\Models\Expansion;
+use Tests\TestCase;
+
+class BlackCardRelationshipTest extends TestCase
```

Comment:
I would suggest that we create a `Models` folder and just have a BlackCardTest file. This way we can have the relationship tests and any tests surrounding accessors in the same place.